lib/cell-manager.coffee

```diff
@@ -0,0 +1,51 @@
+_ = require 'lodash'
+
+module.exports = CellManager =
+    removeAllBreakPoints: ->
```

Comment:
The concern I have is that `cell` is notebook-speak (some users won't be familiar with jupyter notebooks). How about renaming the commands to:
- `hydrogen:run-breakpoint-cell`
- `hydrogen:run-breakpoint-cell-and-move-down`
- `hydrogen:add-breakpoint`
- `hydrogen:remove-all-breakpoints`
- `hydrogen:remove-latest-breakpoint`

I think it's OK to write [breakpoint](http://www.yourdictionary.com/breakpoint) together. I guess the corresponding methods would need renaming.